README.md

```diff
@@ -91,7 +84,12 @@ The constructor for the `PadoNetworkContractClient`.
 
 ***Note:***
 
-> By default, `StorageType` is `ARWEAVE` when `chainName` is `ao`, and `ARSEEDING` when `chainName` is `holesky` or `ethereum`.
+> - By default, `StorageType` is `ARWEAVE` when `chainName` is `ao`, and `ARSEEDING` when `chainName` is `holesky` or `ethereum`.
+>
+> - When using **ARSEEDING** as storage, you need to ***deposit*** a certain amount of ***ETH*** to ***EverPay*** to cover storage, computation, data, and other costs. You can learn more about EverPay at:
```

Comment:
“you need to” can modify to "user need to". User need deposit eth when chain is ethereum.